Library/Homebrew/test/formula_installer_spec.rb

```diff
@@ -14,6 +14,10 @@
     match(&:poured_from_bottle)
   end
 
+  before do
```

Comment:
Any preference about this approach vs. something like `allow(Homebrew).to receive(:args).and_return(some_mock_object)`?